src/components/organisms/TimingEventModal/TimingEventModal.scss

```diff
@@ -16,6 +17,12 @@
     background-color: $light-grey;
   }
 
+  .model-content,
+  .modal-dialog {
+    width: 700px;
+    max-width: 700px;
+  }
+
```

Comment:
Would it be useful to add media query for narrower display? 

Most queries in the project use:
```scss

  @media (max-width: $media-breakpoint--lg) {
   }

  @media (max-width: $media-breakpoint--md) {
  }
```

Reply:
Given we're going to be doing a redesign I'm not sure it's worth putting the effort into making this work on smaller devices. Anything below 700px is going to have a terrible admin site experience at the moment regardless :(